### ISSUE TITLE
Misterious bug with the circuitikz manual rendering

### ISSUE THREAD

**Issue by Rmano** [circuitikz/circuitikz#147]
Look at this screenshot of the manual: 

![image](https://user-images.githubusercontent.com/6414907/52744488-1bab4d80-2fdd-11e9-92f1-90173c6a5c66.png)

but then, if I take the same code out: 
```
\documentclass[border=10pt]{standalone}
%\usepackage[siunitx]{circuitikz}
\usepackage[siunitx]{circuitikzgit}
\begin{document}
\begin{circuitikz} \draw
    (0,0) node[inst amp ra] (opamp) {}
    (opamp.+) node[left] {$v_+$}
    (opamp.-) node[left] {$v_-$}
    (opamp.out) node[right] {out}
    (opamp.up) node[vcc]{}
    (opamp.down) node[vee] {}
    (opamp.refv down) node[ground]{}
    (opamp.refv up) to[short, -o] ++(0,0.3)
    ;
\end{circuitikz}
\end{document}
```

I have the correct connection: 

![image](https://user-images.githubusercontent.com/6414907/52744565-5614ea80-2fdd-11e9-8c09-ee6aa86c1950.png)

Can anyone guess what's happening here? Notice that the rendering of the "up" connection has no sense, given that the circle is in the right position and the line is just "misplaced"...

**Comment by Rmano** [comment]
Ok, now I can reproduce it (it was an old version lingering around). Still I do not know how is this possible...

**Comment by Rmano** [comment]
Fixed by #148
